wfl/cli/cli.py

```diff
@@ -460,18 +461,18 @@ def _castep_eval(ctx, inputs, output_file, workdir_root, directory_prefix, prope
                    "is recPBE with settings tested for radicals")
 @click.option("--orca-additional-blocks", type=click.STRING,
               help="orca blocks to be added, default is None")
-def orca_eval(ctx, inputs, workdir_root, output_file, directory_prefix,
-              orca_command, calc_kwargs, keep_files, output_prefix, scratch_path, n_run, n_hop,
+def orca_eval_bh(ctx, inputs, workdir_root, output_file, directory_prefix,
+              orca_command, calc_kwargs, keep_files, output_prefix, scratchdir, n_run, n_hop,
               orca_simple_input, orca_additional_blocks):
     verbose = ctx.obj["verbose"]
 
-    if scratch_path is not None:
-        if not os.path.isdir(scratch_path):
+    if scratchdir is not None:
```

Comment:
Why is `scratchdir` mandatory here?

Reply:
It isn't mandatory - it defaults to `None` if no argument is given to the command line interface.

Reply:
Yeah, I see now. It's just making sure that _if_ it was passed in, it exists.